config/services.yaml

```diff
@@ -0,0 +1,24 @@
+imports:
+  - { resource: packages/* }
+  - { resource: pimcore/* }
```

Comment:
Do we really need this? Pimcore should import it automatically, right? Also, not sure if we should import the `routing.yaml` into the services.